ui/components/multichain/account-picker/account-picker.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import {
+  ButtonBase,
+  ICON_NAMES,
+  AvatarAccount,
+  AvatarAccountVariant,
+  Text,
+} from '../../component-library';
+import {
+  BackgroundColor,
+  BorderRadius,
+  DISPLAY,
+  FONT_WEIGHT,
+  Size,
+} from '../../../helpers/constants/design-system';
+
+export const AccountPicker = ({ address, name, onClick }) => {
+  const useBlockie = useSelector((state) => state.metamask.useBlockie);
+
+  return (
+    <ButtonBase
+      className="multichain-account-picker"
+      onClick={onClick}
+      backgroundColor={BackgroundColor.transparent}
```

Comment:
Hm. Should ButtonBase be used outside of button? I thought that the only reason ButtonBase existed was to allow shared code between all the buttons?

Reply:
The base `<Button>` component has a load of styles I don't want on this one; this is also a component that DS will be basing a new component off of, so I don't think it needs perfection in the short term.

Reply:
Should we expose a very plain repackaging of ButtonBase under a new moniker "ButtonUnstyled" similar to material/ui's new unstyled exports. I'd really like to keep ButtonBase ONLY as the base for the button components and not used externally from the Button folder. @georgewrmarshall @garrettbear

Reply:
I've updated this PR to use `<Button>` instead of `ButtonBase`.